README.md

```diff
@@ -174,7 +174,7 @@ Below are some commonly encountered issues and possible ways to resolve them. If
 ## The app won't build
 
 - When I run `npm install`, it fails with dependency resolution errors
-  > Sometimes this happens when one or more of the project dependencies gets updated and is out of step with the others. Try running `npm install --legacy-peer-deps` or `npm install --force`.
+  > Sometimes this happens when one or more of the project dependencies gets updated and is out of step with the others. You should always use `npx expo install` command as it also calls for `npm install`.
```

Comment:
Similar to the point above.  Please can you change the last sentence here to:

Try running `npx expo install` and then `npm install --legacy-peer-deps` or `npm install --force`

Reply:
✔️